BritneyS/SparkleSpin
# Player screen

## What you did :question:
This PR includes the preliminary work to build the full 'Choose Players' screen: A navigation controller and a tableview to display a custom cell that includes a custom textfield.

## How to test it :microscope:
1. Check out this branch
2. Run the app
3. On the Home screen, tap the 'Start' button
4. Note that a screen with the title 'Choose Players!' appears with a single text field (see screenshot)
5. Tap the 'Back' button represented the single "chevron" icon (the line arrow that faces left)
6. Note that the Home screen is displayed (see screenshot)

## Any background context you want to provide? :question:
On the 'Choose Players!' screen, I plan to add an 'Add' button that will add rows to the tableview (to enter more names as needed by the user).


## Screenshots (if applicable) :camera:
Not shown: Launch screen
![sparkle-spin-pr-player-screen](https://user-images.githubusercontent.com/8409475/54537152-67e50700-4968-11e9-9d54-b61b52d5e962.gif)
